api/parts/data/fetch.js

```diff
@@ -17,7 +17,9 @@ var fetch = {},
     _ = require('underscore'),
     crypto = require('crypto'),
     usage = require('./usage.js'),
-    plugins = require('../../../plugins/pluginManager.js');
+    STATUS_MAP = require('../jobs/job').STATUS_MAP,
+    plugins = require('../../../plugins/pluginManager.js'),
+    countlyDb = plugins.dbConnection();
```

Comment:
Let's not create new connection, but rather user `common.db` one

Reply:
updated!